test/integration/git-credentials-databag/inspec/git_credentials_databag_spec.rb

```diff
@@ -1,28 +1,24 @@
-require 'serverspec'
-
-set :backend, :exec
-
 describe package 'git' do
   it { should be_installed }
 end
 
 describe file '/root/.git-credentials' do
   it { should be_file }
-  it { should be_owned_by 'root' }
-  it { should be_mode '600' }
+  its('owner') { should be 'root' }
+  its('mode') { should be '0600' }
```

Comment:
This is failing and an incorrect syntax I believe. You have similar issues below as well.

Reply:
Yep, I just found out, fixed it, and came here, then saw your comments :smile: